lib/parallel_tests/tasks.rb

```diff
@@ -7,8 +7,12 @@ def rails_env
         ENV['RAILS_ENV'] || 'test'
       end
 
+      def purge_before_load?
+        ENV.fetch('PURGE_BEFORE_LOAD', 'TRUE').downcase == 'false' ? false : true
+      end
+
       def purge_before_load
-        "db:test:purge" if Gem::Version.new(Rails.version) > Gem::Version.new('4.2.0')
+        "db:test:purge" if purge_before_load? && Gem::Version.new(Rails.version) > Gem::Version.new('4.2.0')
```

Comment:
cannot use Rails.version since this might be used without rails ... `ActiveRecord::Version::STRING` should work